torch/_inductor/codegen/triton.py

```diff
@@ -754,7 +756,9 @@ def __init__(
         self.indirect_max_sizes_printed = {}  # Upper bounds, printed as a string
         self.last_usage = set()
 
-        self.persistent_reduction = self.should_use_persistent_reduction()
+        self.persistent_reduction = (
+            not disable_persistent_reduction
+        ) and self.should_use_persistent_reduction()
```

Comment:
We should update the heuristic for enabling persistent reductions to be less conservative when this flag is set.  Currently it is enabled pretty rarely.